conda/recipes/cuml/meta.yaml

```diff
@@ -14,93 +14,113 @@ package:
 source:
   path: ../../..
 
-build:
-  number: {{ GIT_DESCRIBE_NUMBER }}
-  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
-  script_env:
-    - AWS_ACCESS_KEY_ID
-    - AWS_SECRET_ACCESS_KEY
-    - AWS_SESSION_TOKEN
-    - CMAKE_C_COMPILER_LAUNCHER
-    - CMAKE_CUDA_COMPILER_LAUNCHER
-    - CMAKE_CXX_COMPILER_LAUNCHER
-    - CMAKE_GENERATOR
-    - PARALLEL_LEVEL
-    - SCCACHE_BUCKET
-    - SCCACHE_IDLE_TIMEOUT
-    - SCCACHE_REGION
-    - SCCACHE_S3_KEY_PREFIX=cuml-aarch64 # [aarch64]
-    - SCCACHE_S3_KEY_PREFIX=cuml-linux64 # [linux64]
-    - SCCACHE_S3_USE_SSL
-  ignore_run_exports_from:
-    - {{ compiler('cuda') }}
-    {% if cuda_major != "11" %}
-    - cuda-cudart-dev
-    {% endif %}
-    - cuda-python
+outputs:
```

Comment:
I haven't verified this recipe works at all, just based it on other recipes I've done in the past that have used multiple outputs for metapackages. Hoping CI/review will catch any issues here 🤞 .